### PR TITLE
Fab 27 front end

### PR DESCRIPTION
**Pull request by D22124491-NaveenMaheswaran**
routes
componenets

### CODE REVIEW

**Review by UmamaSumlinTasnuva — APPROVED** [review]
All the routes are properly written. I like the way you have divided our system into several components. It's very easy to navigate all the files for each of the component. Just in your next change, add a component for the page of previewing the data after user uploads it. Keep up the good work @D22124491-NaveenMaheswaran

File: Frontend-Angular/data-polish-frontend/src/app/app.component.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/app.component.ts:1]: Consider adding error handling to improve robustness

File: Frontend-Angular/data-polish-frontend/src/app/app.module.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/app.module.ts:1]: Consider removing redundancy in route setups and considering a catch-all route for unmatched paths.

File: Frontend-Angular/data-polish-frontend/src/app/data-profile/data-profile.component.spec.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/data-profile/data-profile.component.spec.ts:1]: LG

File: Frontend-Angular/data-polish-frontend/src/app/data-profile/data-profile.component.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/data-profile/data-profile.component.ts:1]: LG

File: Frontend-Angular/data-polish-frontend/src/app/home-page/home-page.component.spec.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/home-page/home-page.component.spec.ts:1]: In order to enhance the robustness and value of the tests, consider adding more test cases that evaluate the component's functionality, interactions, and side effects. Also, ensure that any dependencies are adequately provided or mocked.

File: Frontend-Angular/data-polish-frontend/src/app/home-page/home-page.component.ts
Comment (OluwatobiOmole) [inline Frontend-Angular/data-polish-frontend/src/app/home-page/home-page.component.ts:1]: LG

**Review by OluwatobiOmole — APPROVED** [review]
Code sections reviewed and suggestions given

**Pull request merged**